server/sqlstore/playbook.go

```diff
@@ -152,28 +151,34 @@ func (p *playbookStore) Get(id string) (out playbook.Playbook, err error) {
 }
 
 // GetPlaybooks retrieves all playbooks that are not deleted.
-func (p *playbookStore) GetPlaybooks() (out []playbook.Playbook, err error) {
+func (p *playbookStore) GetPlaybooks() ([]playbook.Playbook, error) {
 	tx, err := p.store.db.Beginx()
 	if err != nil {
-		return out, errors.Wrap(err, "could not begin transaction")
+		return nil, errors.Wrap(err, "could not begin transaction")
 	}
 	defer p.store.finalizeTransaction(tx)
 
-	err = p.store.selectBuilder(tx, &out, p.playbookSelect.Where(sq.Eq{"DeleteAt": 0}))
+	var out []playbook.Playbook
+	err = p.store.selectBuilder(tx, &out, p.store.builder.
+		Select("ID", "Title", "Description", "TeamID", "CreatePublicIncident", "CreateAt",
+			"DeleteAt", "NumStages", "NumSteps").
+		From("IR_Playbook AS p").
+		Where(sq.Eq{"DeleteAt": 0}))
+
 	if err == sql.ErrNoRows {
-		return out, errors.Wrap(playbook.ErrNotFound, "no playbooks found")
+		return nil, errors.Wrap(playbook.ErrNotFound, "no playbooks found")
```

Comment:
Apologies if I've asked this in a previous PR and lost the context, but why is this an error?
```suggestion
return nil, nil
```

Reply:
It was for code that was doing logic when it received ErrNotFound. But really, we don't even use this function, except in the self-test and the testing code, so we can basically remove it if you want. 
I prefer to keep it -- it caught a nasty (self-inflicted) bug today, since it's the only place where multiple playbooks are returned with memberIDs.